js/src/personal/kelly/Kelly.module.css

```diff
@@ -1,26 +1,9 @@
-.body_container{
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  height: 50vw;
-  background-color: #fdf7ee;
-  background-image: url("https://i.pinimg.com/564x/8a/28/bd/8a28bd8947deb747aba8b2557fb41982.jpg");
-  background-position: center;
-  background-repeat: no-repeat;
+html{
```

Comment:
Your commit messages should be cleaned up a bit.
Wouldn't mind letting you ship for now if you remove the html section here, as it effects other people pages as it's global.

Reply:
removed the html element for now